src/main/osd/osd_elements.c

```diff
@@ -1858,6 +1864,7 @@ void osdElementsInit(bool backgroundLayerFlag)
 {
     backgroundLayerSupported = backgroundLayerFlag;
     activeOsdElementCount = 0;
+    pt1FilterInit(&batteryEfficiencyFilt, pt1FilterGain(EFFICIENCY_CUTOFF_HZ, 1.0f/50.0f));
```

Comment:
In `pt1FilterGain()`: 
The first argument needs to be the looptime of the `osdElementEfficiency()` function and `EFFICIENCY_CUTOFF_HZ` needs to  be the second argument.

Reply:
I corrected the dt.  However the arguments are in the right order.

float pt1FilterGain(float f_cut, float dT)

Reply:
Yeah sorry my bad. I corrected my comment.